upi/openstack/post-install.yaml

```diff
@@ -2,13 +2,6 @@
   gather_facts: no
 
   tasks:
-  - name: 'Create snat policy file'
-    template:
-      src: cluster_snat_policy.conf.j2
-      dest: cluster_snatpolicy.yaml
-
-  - name: 'Create cluster SNAT policy'
-    k8s:
-      state: present
-      kubeconfig: "{{ aci_cni['kubeconfig'] }}"
-      src: cluster_snatpolicy.yaml
+  - name: 'Replace cluster endpoint stratergy to Loadbalancer'
```

Comment:
This should have ideally been: Set clusterEndpointStrategy to ...

Reply:
resolved